docs/source/execution_device_interoperability.ipynb

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
<p>We probably do not want to encourage introspection of <code>global_settings</code>. If we want something like this, let's introduce a <code>get_current_device_type</code> or something like that. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuml/pull/5130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>